x-pack/plugins/enterprise_search/public/applications/app_search/components/result_field_value/result_field_value.scss

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+.enterpriseSearchDataType {
+  &--number,
+  &--float {
+    color: $euiColorAccentText;
+    font-family: $euiCodeFontFamily;
+  }
+
+  &--location {
+    color: $euiColorSuccessText;
```

Comment:
Did I get confused on this? I thought this was the `secondary` color before, not `success`? Not a blocker though
```suggestion
    color: $euiColorSecondaryText;
```

Reply:
Ohhhh, did I mess this up 🤔 

<img width="1116" alt="colors" src="https://user-images.githubusercontent.com/1427475/99968499-33456780-2d67-11eb-9031-001d67ca7b00.png">

I used the value from this page. Hrmmm.

Reply:
Oh I guess it's just a naming then, no worries!